torchsynth/module.py

```diff
@@ -721,7 +745,7 @@ def __init__(
                 )
             )
 
-        super().__init__(synthglobals=synthglobals, device=device, **kwargs)
+        super().__init__(synthglobals=synthglobals, **kwargs)
```

Comment:
okay nice pattern

Reply:
IMHO TBH then we can also remove SynthGlobals from each init since it's in kwargs

Reply:
sure I dig that

Reply:
pulling back on this one. synthglobals is a required positional argument in SynthModule so it's actually not in kwargs in that init.

Reply:
Right but just because it's a required kwarg, if a submodule has it in kwargs should be fine no?

Reply:
ahhh ya I understand, yes we can get rid of the synthglobals from the super init.